mdpow/templates/system_octwet.top

```diff
@@ -0,0 +1,30 @@
+; system.top
+; Part of the POW package.
+; Copyright (c) 2010-2011 Oliver Beckstein and Bogdan Iorga
+; Template for simple Gromacs simulations of a compound in water
+; NOTE: This file is processed with mdpow.equil.topology()
+;       which expects EXACTLY this file as input in order to be able
+;       to change it as needed. See the source mdpow/equil.py for details.
+
+; Include forcefield parameters
+#include "oplsaa.ff/forcefield.itp"
+
+; Include compound topology
+#include "compound.prm"
+
+#include "compound.itp"
+
+; Include solvent topology
+#include "oplsaa.ff/tip4p.itp"
+#include "water.itp"
+
+; Include topology for OPLS/AA ions 
+#include "oplsaa.ff/ions_opls.itp"
+
+[ system ]
+; Name
+Compound DRUG in solvent (OPLS-AA parameters)
+
+[ molecules ]
+; Compound        #mols
+DRUG               1
```

Comment:
The old version GROMACS needs one line `OcOH                   1` after the `DRUG` line here.

Reply:
We can use Gromacs [preprocessor directives](http://manual.gromacs.org/current/onlinehelp/gmx-grompp.html) in the file and define a variable `MDPOWPREGMX2019`
```
DRUG               1
#ifdef MDPOWPREGMX2019
OcOH               1
#endif
```

You would then add to the [MDP file](http://manual.gromacs.org/current/user-guide/mdp-options.html)
```
define  = -DMDPOWPREGMX2019
```

In this way, we have the same top file and just switch how it is read by Gromacs depending on the version. MDPOW will have to add the appropriate *define* option when it generates the MDP file(s).

Reply:
I *think* that this will work but perhaps try it out manually first.